USAGE.md

```diff
@@ -163,6 +163,19 @@ dependencies:
 
 It will be a dependency violation when `components/shop_identity` tries to reference a constant that is not within `components/platform` or itself.
 
+If you'd like to enforce some, but not all, dependencies, you can use the `dependencies` key. For example, if we want to enforce dependencies from `components/shop_identity` onto `components/platform`, but not onto `components/admin`, we can do this:
+
+```yaml
+# components/shop_identity/package.yml
+enforce_dependencies: true
+dependencies:
+  - components/platform
+ignored_dependencies:
```

Comment:
I feel like `temporary_dependencies` dependencies would be a better name. Ignore seems like it is on to depend on this code and nothing needs to be done in the future, when in reality you want those to be fixed, but you can't right now.

```suggestion
temporary_dependencies:
```

Reply:
Would `currently_ignored_dependencies` satisfy both your viewpoints? (And be a sufficiently obvious awkward and long name that folks get the message that they shouldn't want this?)

Reply:
> Another possibility, if that we also record circular dependencies in the todo, instead of introducing a way to ignore dependencies.

🤔 This is a very compelling idea. `package_todo.yml` already tracks dependency violations, why not circular dependency violations as well? It could get noisy but surfacing circular dependencies in the PR diff could be very illuminating and force teams to acknowledge this risk/tech debt they are taking on by having them.

@alexevanczuk would this address your needs for ignored dependencies?

Reply:
It's pretty close, because it allows us to ignore dependencies by listing them once. I still really like the communication vehicle of it being "ignored" because it allows folks to express "these may or may not be our long-term dependencies: we don't want to consider them for violations" as a separate expressive statement compared to "these are our true dependencies." Having them combined lets folks ignore them more easily by we'd still end up with the conceptual issue of `dependencies` being overloaded.